Documentation/api-reference.md

```diff
@@ -0,0 +1,138 @@
+# Adobe Experience Platform Identity for Edge Network Extension - Android
+
+## Prerequisites
+
+Refer to the [Getting Started Guide](getting-started.md)
```

Comment:
Where is the getting-started.md?

Reply:
It's there in getURLVariables feature branch